examples/4_train_policy_with_script.md

```diff
@@ -0,0 +1,157 @@
+This tutorial will explain the training script, how to use it, and particularly the use of Hydra to configure everything needed for the training run.
+
+## The training script
+
+LeRobot offers a training script at [`lerobot/scripts/train.py`](../../lerobot/scripts/train.py). At a high level it does the following:
+
+- Loads a Hydra configuration file for the following steps (more on Hydra in a moment).
+- Makes a simulation environment.
+- Makes a dataset corresponding to that simulation environment.
+- Makes a policy.
+- Runs a standard training loop with forward pass, backward pass, optimization step, and occasional logging, evaluation (of the policy on the environment), and checkpointing.
+
+## Our use of Hydra
+
+Explaining the ins and outs of [Hydra](https://hydra.cc/docs/intro/) is beyond the scope of this document, but here we'll share the main points you need to know.
+
+First, consider that `lerobot/configs` might have a directory structure like this (this is the case at the time of writing):
```

Comment:
```suggestion
First, `lerobot/configs` has a directory structure like this:
```

Reply:
I'd like this to be more concrete as well. But being a little vague allows room for error as this is a very not-maintainable bit of documentation.

@aliberts do you think we can use a tool like this in CI? https://github.com/UmbrellaDocs/linkspector

Reply:
@Cadene I made my sentence less apologetic, leaving just one "might".

Reply:
I dont think we should use "might". We should tell exactly what is expected. No?

Reply:
Did might -> will (everywhere), and used "like" or "something like". There may be minor refactors that have no bearing on the essentials that this document tries to convey, so I don't want the language to overly-commit.